mule-extensions-api/src/main/java/org/mule/runtime/extension/api/stereotype/ImplicitStereotypeDefinition.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.stereotype;
+
+import static java.util.Objects.hash;
+import static java.util.Optional.ofNullable;
+
+import java.util.Optional;
+
+/**
+ * A special kind of stereotype definition, the model from which it is created will depend of the type that contains this
+ * definition.
+ *
+ * @since 1.3
+ */
+public class ImplicitStereotypeDefinition implements StereotypeDefinition {
```

Comment:
shouldn't this also define the extension's namespace? check the base interface foor default methods

Reply:
the namespace is set by the enricher, as per the javadoc in `StereotypeDefinition#getNamespace`